src/test/scala/chiselTests/BoringUtilsSpec.scala

```diff
@@ -48,7 +48,8 @@ class BoringUtilsSpec extends ChiselFlatSpec with ChiselRunners {
 
   behavior.of("BoringUtils.{addSink, addSource}")
 
-  it should "connect two wires within a module" in {
+  //TODO: SFC->MFC, this test is ignored tests rely on unsupported custom annotations
```

Comment:
I thought MFC supported `BoringUtils` ? Maybe it was in a later version than the one tested?